addons/web/static/src/webclient/navbar/navbar.scss

```diff
@@ -3,6 +3,10 @@
 // ============================================================================
 .o_main_navbar {
     @include print-variable(o-navbar-height, $o-navbar-height);
+    --NavBar-entry-color--active: #{$o-white};
```

Comment:
I don't understand this :/
We have a `$o-navbar-entry-color--active` **SCSS** variable that could be used as default/fallback.
I think we should remove this line and, wherever is needed, do something like:

```scss
color: var(--NavBar-entry-color--active, #{$o-navbar-entry-color--active}
```